### ISSUE TITLE
Unexpected behavior on remove network

### ISSUE THREAD

**Issue by abefernan** [iov-one/ponferrada#961]

**Comment by annatorok** [comment]
Yes, eg. you cannot add Lisk and there is no "Add more" option either

![Screenshot 2020-01-17 at 12 31 43](https://user-images.githubusercontent.com/7419692/72609641-c337ab00-3925-11ea-9833-bc51dcb9eaca.png)
